src/Docker.DotNet/Endpoints/ContainerOperations.cs

```diff
@@ -254,6 +256,21 @@ public Task<Stream> GetContainerLogsAsync(string id, ContainerLogsParameters par
             return this._client.MakeRequestForStreamAsync(new[] { NoSuchContainerHandler }, HttpMethod.Get, $"containers/{id}/logs", queryParameters, null, cancellationToken);
         }
 
+        public async Task GetContainerLogsAsync(string id, ContainerLogsParameters parameters, CancellationToken cancellationToken, IProgress<string> progress = null)
```

Comment:
Does this compile? I would assume that for this transfer period what we will need to do is to make the progress a non default parameter. Then we will have 2 physical overloads and then when I merge/remove the [Obsolete] version I will make it optional again to keep the same set of overloads.